http/stats.md

```diff
@@ -6,24 +6,32 @@ All times (latencies) are measured in float64 fractions of microseconds.
 
 There is no special support for multi-part HTTP requests and responses. These are just treated as one request.
 
+## Units
+
+* Latencies are measures in float64 milliseconds, denoted "ms"
+* Sizes are measured in bytes, denoted "By"
```

Comment:
I think "By" might be confused with "by". How about using only "B" to denote "byte"?

Reply:
These are from http://unitsofmeasure.org/ucum.html (also used by Stackdriver)

Reply:
Added a note to this effect

Reply:
I see, thanks!